tests/test_sites.py

```diff
@@ -135,8 +139,12 @@ def setup(self, db, settings):
         is_multisite = figures.helpers.is_multisite()
         assert is_multisite
         self.site = SiteFactory(domain='foo.test')
-        self.organization = OrganizationFactory(sites=[self.site])
-        assert Site.objects.count() == 2
+        self.organization = OrganizationFactory()
+        self.edly_sub_organization = EdlySubOrganizationFactory(
+            lms_site=self.site,
+            edx_organization=self.organization
+        )
+        assert Site.objects.count() == 3
```

Comment:
add  a comment why site's count is 3 now

Reply:
Should I add a code comment or a comment here?
This count is 3 as the `EdlySubOrganizationFactory` creates a new studio site as well.